testing/skim/APKBUILD

```diff
@@ -45,42 +45,75 @@ package() {
 
 	install -Dm644 plugin/skim.vim -t "$pkgdir"/usr/share/vim/vimfiles/plugin
 
-	install -Dm644 shell/completion.bash "$pkgdir"/usr/share/bash-completions/completions/sk
-	install -Dm644 shell/completion.zsh "$pkgdir"/usr/share/zsh/site-finctions/_sk
+	install -Dm644 shell/completion.bash "$pkgdir"/usr/share/bash-completion/completions/sk
+	install -Dm644 shell/completion.zsh "$pkgdir"/usr/share/zsh/site-functions/_sk
 
 	install -Dm644 shell/key-bindings.* -t "$pkgdir"/usr/share/"$pkgname"
 }
 
 vim() {
-	install -d "$subpkgdir"/usr/share
+	depends=""
+	pkgdesc="Vim plugin for $pkgname"
+	install_if="$pkgname=$pkgver-r$pkgrel vim"
+
+	mkdir -p "$subpkgdir"/usr/share
 	mv "$pkgdir"/usr/share/vim "$subpkgdir"/usr/share
 }
 
 tmux() {
-	install -d "$subpkgdir"/usr/bin
+	depends="bash"
```

Comment:
the tmux script [requires bash](https://github.com/lotabout/skim/blob/787f173b0bbb38da8920792bb6cf0eebc421768b/bin/sk-tmux#L1)